os/net/lwip/src/core/ipv4/dhcps.c

```diff
@@ -406,7 +412,14 @@ static void dhcps_send_msg(struct dhcps_msg *m, u8_t msg_type)
 		return;
 	}
 
-	if (udp_sendto(pcb_dhcps, p, &broadcast_dhcps, DHCP_CLIENT_PORT) != ERR_OK) {
+	ip_addr_t ip_temp = IPADDR4_INIT(0x0);
```

Comment:
perhaps use a different variable name, like `ip_host`, instead of `temp`.

Reply:
it's temporary variable to convert ip address so I think it's ok